### DISCUSSION TITLE
Use req in OpenResty

### DISCUSSION THREAD

**Discussion by kingluo** [imroc/req#227] [General]
Here is my open-source project:

https://github.com/kingluo/lua-resty-ffi-req

It encapsulates req via [lua-resty-ffi,](https://github.com/kingluo/lua-resty-ffi) so that you could utilize req to send requests in OpenResty.

Here is the blog post:

http://luajit.io/posts/http3-client-for-openresty

Any suggestions and discussions are welcome! Thanks!

@imroc If you're interested and have time, please look at my [benchmark frame graph.](http://luajit.io/posts/http3-client-for-openresty/pprof_cpu.svg) It seems that golang uses a lot of CPU than expected. Looking forward to your advice, thanks!

**Comment by imroc** [comment]
Nice project, but I don't have time recently, I can take a look if I'm free in the future

**Reply by kingluo** [reply]
Thank you very much!
